packages/components/notification/src/index.vue

```diff
@@ -100,7 +102,13 @@ export default defineComponent({
 
     const typeClass = computed(() => {
       const type = props.type
-      return type && TypeMap[type] ? `el-icon-${TypeMap[type]}` : ''
+      return type && TypeComponentsMap[props.type]
+        ? `el-notification--${type}`
+        : ''
+    })
+
+    const iconComponent = computed(() => {
+      return props.icon || TypeComponentsMap[props.type] || ''
```

Comment:
`TypeComponentsMap[props.type] `should have higher priority than` props.icon`

Reply:
updated